src/sentry/eventtypes/base.py

```diff
@@ -55,15 +55,13 @@ class BaseEvent:
     id = None
 
     def get_metadata(self, data):
-        metadata = {}
+        metadata = self.extract_metadata(data)
```

Comment:
The changes to this function are a pure refactor, right? (Which is fine, I'm just wondering if I'm missing some functional change here).

Reply:
sry yeah this is just refactor